### PR TITLE
Remove collators from bootnodes list

### PR DESCRIPTION
**Pull request by BulatSaif**
We are no longer using collators in the bootnode list; instead, we have replaced them with "connect" nodes. This change serves two purposes: first, it removes unnecessary load from collators, and secondly, it enables us to proceed with the planned decommission of collators nodes as outlined in [Referendum 84](https://polkadot.subsquare.io/referenda/referendum/84)

### CODE REVIEW

File: parachains/chain-specs/bridge-hub-kusama.json
Comment (joepetrowski) [inline parachains/chain-specs/bridge-hub-kusama.json:1]: This is Kusama, but the referendum is for Polkadot. Did you mean to make this change to `bridge-hub-polkadot` (instead of in addition)?

Comment (BulatSaif) [reply parachains/chain-specs/bridge-hub-kusama.json:1]: Collators from bridge-hub-polkadot were already removed (I believe I did it before the release). This cleanup is for the remaining parachains.

Comment (BulatSaif) [reply parachains/chain-specs/bridge-hub-kusama.json:1]: Note: For testnet parachains, we still use collators as bootnodes since the load is smaller, and it doesn't make financial sense to spin up additional nodes just for bootnodes.

**Review by joepetrowski — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by BulatSaif** [comment]
> Error: Statuses failed for [c86fa1d](https://github.com/paritytech/cumulus/commit/c86fa1d4ee7819c5e1212226f5fe760a6da99a82)

Rule "Core developers" needs at least 2 approvals, but 1 were given

**Review by bkontur — APPROVED** [review]

**Comment by BulatSaif** [comment]
bot merge

**Pull request merged**
